Fixed blank spell entities

Frost2779 opened the pull request:
Fixed spell entries related to the removal of:
Booming Blade
Green-Flame Blade
Sword Burst
Phantasmal Force

Fixed issue #42

Frost2779 committed:
Fixed blank spell entities 

Fixed spell entries related to the removal of:
Booming Blade
Green-Flame Blade
Sword Burst
Phantasmal Force

Demonsthere commented:
Hi There and thanks for the contribution! 
I would prefer we remove the non SRD spells from the code rather than commenting it.

Frost2779 committed:
Removed all non-SRD spells

I believe I was able to remove all non-SRD spells from all classes. According to my IDE I don't have any syntax errors, but I have yet to successfully setup a Dev enviroment to actually test the changes. I believe it should work all fine and dandy but It would be appreciated if someone could double check for me. I have made all the changes following https://5thsrd.org/ as far as what is allowed and what isn't.
